platform/InternalDateTime.roc

```diff
@@ -6,7 +6,7 @@ interface InternalDateTime
     ]
     imports []
 
-DateTime : { year : U128, month : U128, day : U128, hours : U128, minutes : U128, seconds : U128 }
+DateTime : { year : I128, month : I128, day : I128, hours : I128, minutes : I128, seconds : I128 }
```

Comment:
Not from this PR, but just curious, do we need these to also be I128s, can they be something way smaller?

Reply:
That thought crossed my mind too... those are huge types for values that will never approach the bounds of their type...